x-pack/plugins/synthetics/common/rules/types.ts

```diff
@@ -11,7 +11,8 @@ import type {
   PagerDutyConnectorTypeId,
   ServerLogConnectorTypeId,
   ServiceNowITSMConnectorTypeId as ServiceNowConnectorTypeId,
-  SlackConnectorTypeId,
+  SlackApiConnectorTypeId,
```

Comment:
Looks like we will need to add support for the new connector type, given this change, here: https://github.com/elastic/kibana/blob/main/x-pack/plugins/synthetics/common/rules/alert_actions.ts ; otherwise, I wonder if they'd get some kind of chooser for a connector type, pick the new one (somehow), but then the code in that `alert_actions.ts` would not do the right thing ...